druid/src/optics/lens/lens.rs

```diff
@@ -35,13 +37,13 @@ use crate::Data;
 /// sophistication, for example combinators to combine lenses.
 ///
 /// [Haskell lens]: http://hackage.haskell.org/package/lens
-pub trait Lens<T: ?Sized, U: ?Sized> {
+pub trait Lens<S: ?Sized, A: ?Sized> {
```

Comment:
What is the goal of these renames?

Reply:
This is to match more closely how they are named in external resources, as in [haskell](https://hackage.haskell.org/package/lens-4.19.2/docs/Control-Lens-Lens.html#t:Lens-39-), [Scala](https://www.optics.dev/Monocle/api/monocle/Lens$.html) and in some blog posts/videos.. I'm not sure what they mean: I found something like "Source" and "Target", which doesn't makes sense to me. But overall, I think this terminology is very adopted in external resources.

But I didn't make it entirely consistent, as I ended up preserving the A -> B -> C terminology in some cases, where multiple lenses are used.

I applied the renames to the prisms as well (as I initially copied the original lens terminology).

Reply:
Recently I dropped all `T`-`U`, `S`-`A` and `A`-`B`-`C` terminology and just stuck with `T1`-`T2` and `T1`-`T2`-`T3` as I think it becomes easier to read all of the related functions. The exception is where only `T` is used.